sparql/sparql12/syntax-escaping/manifest.ttl

```diff
@@ -0,0 +1,68 @@
+@prefix rdf:    <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix :       <https://w3c.github.io/rdf-tests/sparql/sparql12/escaping#> .
+@prefix rdfs:   <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix mf:     <http://www.w3.org/2001/sw/DataAccess/tests/test-manifest#> .
+@prefix qt:     <http://www.w3.org/2001/sw/DataAccess/tests/test-query#> .
+@prefix dawgt:  <http://www.w3.org/2001/sw/DataAccess/tests/test-dawg#> .
+
+:manifest  rdf:type mf:Manifest ;
+    rdfs:label "SPARQL Codepoint-Escaping Tests" ;
```

Comment:
We should indicate that these are related to the character stream processing. Calling them "escaping" is too general.

Suggestion:
1. Rename `syntax-escaping/` as `syntax-char-stream-processing/`
2. `rdfs:label "Character Stream Processing Tests" ;`

Reply:
@afs I'll rename as requested, but unsure about the rdfs:label change. "Codepoint Escape" is a term used in the spec, but I don't think we use anything similar to "Character Stream Processing". I'm obviously biased as the author here, but I think without context I'd be confused by what "Character Stream Processing Tests" were, but would have a pretty good idea about "Codepoint-Escaping Tests". Thoughts?